scripts/cli/lec.sh

```diff
@@ -112,6 +112,7 @@ _printHelpAndExit() {
 		  stop                             Stop a Composer project
 		  clean                            Stop a Composer project and remove Docker volumes
 		  exportData                       Export container data for a Composer project
+		  list                             List entities of a various types
```

Comment:
let's show this as:
```
list [<entity>]
```
to indicate than an entity can _optionally_ (the `[...]` part) be passed to the command as a _positional parameter_ (the <...> part).

Reply:
http://docopt.org/ is a pretty good resource for this kind of thing.